wdl_runner/wdl_runner.sh

```diff
@@ -7,27 +7,36 @@
 # https://developers.google.com/open-source/licenses/bsd
 
 set -o errexit
-set -o nounset
 
 readonly INPUT_PATH=/pipeline/input
 
 # WDL, INPUTS, and OPTIONS filenames are all passed into
 # the pipeline as environment variables - write them out as
 # files.
 mkdir -p "${INPUT_PATH}"
-cp "${WDL}" "${INPUT_PATH}/wf.wdl"
-cp "${WORKFLOW_INPUTS}" "${INPUT_PATH}/wf.inputs.json"
-cp "${WORKFLOW_OPTIONS}" "${INPUT_PATH}/wf.options.json"
+
+# Only pass in the argument if it exists
```

Comment:
While this is pretty cool, I actually like the previous implementation better: it's much easier to read and maintain (the new one requires the reader to keep a mental map of args and their order in three separate variables). Perhaps you can revert it back to the original form and add just add a condition for `WORKFLOW_DEPENDENCIES` that you can then add to your `extraArgs` variable.